lib/src/message/trailer.rs

```diff
@@ -32,6 +39,12 @@ impl TrailerValue {
         }
     }
 
+    pub fn append(self, slice: &str) -> TrailerValue {
```

Comment:
Maybe a comment that this automatically turns everything into a `TrailerValue::String`?